examples/dingo-gov-lens/.env.example

```
@@ -0,0 +1,13 @@
+POSTGRES_DB=dingo_metadata
+POSTGRES_USER=dingo
+POSTGRES_PASSWORD=dingo
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)